test/specs/parsing/axes/ParentAxis.tests.ts

```diff
@@ -87,8 +87,8 @@ describe('parent', () => {
 				getParentNode(node: Node, _bucket?: string | null): Node | null {
 					return node.parentNode;
 				},
-				getChildNodes(node: Node, _bucket?: string | null): Node[] {
-					return node.childNodes;
+				getFirstChild: (node, bucket: string | null) => {
+					return null;
 				},
```

Comment:
nit: might be better to write this as a method instead of an arrow function for consistency?